### DISCUSSION TITLE
Registro de domínios

### DISCUSSION THREAD

**Discussion by ericdouglas** [frontendbr/forum#301] [Discussão]
1. Onde você tem costume de registrar seus domínios?
2. Quais fatores você considera na escolha do local para registrar um novo domínio?

**Comment by kazzkiq** [comment]
**1. Onde você tem costume de registrar seus domínios?**
[Registro.br](http://registro.br/) para domínios nacionais, [Godaddy](https://br.godaddy.com/) para todos os outros.

**2. Quais fatores você considera na escolha do local para registrar um novo domínio?**
Basicamente preço e facilidade de pagamento/administração. No caso do Registro.br, além de geralmente cobrar o mesmo preço (ou mais barato) que empresas de hospedagem, ele ainda te dá a vantagem de contratar o domínio "direto da fonte" ([NIC.br](http://nic.br/)), ao invés de ficar dependendo de hospedagens administrando seu domínio (o que pode ser uma dor de cabeça caso você queira reivindicar ele para trocar de hospedagem, etc).

No caso de domínios internacionais, a Godaddy sempre teve preços competitivos e a forma de pagamento é muito tranquila, é só sair fugindo dos inúmeros checkboxes no meio do processo te oferecendo hospedagem, anonimato, e-mail, e certificados SSL que tá tudo certo. 😄 

Ah, ainda sobre a Godaddy, fica uma dica: **NUNCA** deixe seu domínio expirar lá, ao invés de congelá-lo como o Registro.br faz, eles automaticamente reivindicam o domínio e colocam ele numa espécie de "leilão" que geralmente já começa com valores beeem acima do que você pagou por ele (ex.: você pagou `R$30,00` e o leilão começa a `R$150,00`).

**Comment by brunoravanhani** [comment]
Eu geralmente uso Godaddy também, mas realmente é um problema se deixar expirar o domínio.

**Comment by ramonsenadev** [comment]
Eu costumo utilizar o namecheap para registro de qualquer domínio que não seja .com.br, o preço é bem acessível e o atendimento é muito bom!

**Comment by rubenmarcus** [comment]
namecheap eu usava até o dolar ficar mais que 3 reais, quando o dolar estava 2, ou menos era muito mais vantajoso. .com a 20 reais... 

Agora não é tão vantajoso assim

**Comment by theandersonn** [comment]
**1. Onde você tem costume de registrar seus domínios?**
Nacionais -> Registro.br
Outros -> Name.com - Descobri essa empresa e tive uma bela experiência, são ágeis e sem burocracia. A administração de dns é super de boa também.  

**2. Quais fatores você considera na escolha do local para registrar um novo domínio?**
Agilidade no atendimento
Transparência nas operações (_tem empresas de hospedagens nacionais que tentam travar o dns_)
Preço

**Comment by felipeuntill** [comment]
Reitero as afirmações do @kazzkiq e de adendo:

**Quais fatores você considera na escolha do local para registrar um novo domínio?**
Jamais registre seu domínio em intermediários, já cheguei a perder dominios de aplicações em produção porque o dominio fora comprado na empresa de hospedagem e a respectiva faliu.
Portanto pra mim registro de dominio se restringe a [GoDaddy](https://br.godaddy.com/) e [registro.br](http://registro.br/).

**Comment by rubenmarcus** [comment]
Registro.br é muito burocratico. Perdi um email, que eu tinha dominios muito antigos, pra fazer verificação de senha tem que enviar uma carta pros caras..

**Comment by kazzkiq** [comment]
@rubenmarcus hahaha isso também já aconteceu comigo! Quando li que precisava mandar uma carta via Correios mesmo, quase caí pra trás da cadeira. O pior é que nem lembro se cheguei a enviar a carta ou se acabei recuperando o cadastro na base da adivinhação do login mesmo... 😄
